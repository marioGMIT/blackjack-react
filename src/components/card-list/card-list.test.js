import React from 'react';
import { shallow } from 'enzyme';

import CardList from './card-list.component'

describe('CardList component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CardList label="Hit"/>);
  });

  it('should render CardList component', () => {
    expect(wrapper).toMatchSnapshot();
  });

});