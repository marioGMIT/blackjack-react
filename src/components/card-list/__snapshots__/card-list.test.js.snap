// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList component should render CardList component 1`] = `ShallowWrapper {}`;
